File: app/src/main/java/com/biryanistudio/todo/TodoApplication.java
package com.biryanistudio.todo;

import android.app.Application;

import com.crashlytics.android.Crashlytics;
import com.squareup.leakcanary.LeakCanary;

import io.fabric.sdk.android.Fabric;

/**
 * Created by Aakaash Jois on 11-11-2016 at 12:20 AM.
 */

public class TodoApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Fabric.with(this, new Crashlytics());
        if (LeakCanary.isInAnalyzerProcess(this)) return;
        LeakCanary.install(this);
    }
}

File: app/src/main/java/com/biryanistudio/todo/TodoApplication.kt
package com.biryanistudio.todo

import android.app.Application
import io.realm.Realm

/**
 * Created by aakaashjois on 23/06/17
 */

class TodoApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Realm.init(this)
    }
}